sql/src/main/java/org/apache/druid/sql/calcite/aggregation/builtin/EarliestLatestAnySqlAggregator.java

```diff
@@ -340,5 +345,47 @@ private static class EarliestLatestSqlAggFunction extends SqlAggFunction
           Optionality.FORBIDDEN
       );
     }
+
+    @Override
+    public SqlNode rewriteCall(
```

Comment:
I wonder if after this patch constructors of classes like `LongLastAggregatorFactory` should still accept `timeColumn` as `null` or not

Reply:
Good point! I think it shouldn't. Maybe can take up those modifications in a separate PR.